x/emissions/types/params.go

```diff
@@ -37,16 +36,21 @@ func DefaultParams() Params {
 		MaxTopForecastersToReward:       uint64(6),                                 // max this many top forecasters by score are rewarded for a topic
 		MaxTopReputersToReward:          uint64(12),                                // max this many top reputers by score are rewarded for a topic
 		CreateTopicFee:                  cosmosMath.NewInt(10),                     // topic registration fee
-		SigmoidA:                        alloraMath.NewDecFromInt64(8),             // sigmoid function parameter, a = 8
-		SigmoidB:                        alloraMath.MustNewDecFromString("0.5"),    // sigmoid function parameter, b = 0.5
 		MaxRetriesToFulfilNoncesWorker:  int64(1),                                  // max throttle of simultaneous unfulfilled worker requests
 		MaxRetriesToFulfilNoncesReputer: int64(3),                                  // max throttle of simultaneous unfulfilled reputer requests
 		RegistrationFee:                 cosmosMath.NewInt(6),                      // how much workers and reputers must pay to register per topic
-		DefaultPageLimit:                uint64(100),                               // default limit for pagination
+		DefaultPageLimit:                uint64(100),                               // how many topics to return per page during churn of requests
 		MaxPageLimit:                    uint64(1000),                              // max limit for pagination
 		MinEpochLengthRecordLimit:       int64(3),                                  // minimum number of epochs to keep records for a topic
 		MaxSerializedMsgLength:          int64(1000 * 1000),                        // maximum size of data to msg and query server in bytes
 		BlocksPerMonth:                  uint64(525960),                            // ~5 seconds block time, 6311520 per year, 525960 per month
+		PRewardInference:                alloraMath.NewDecFromInt64(1),             // fiducial value for rewards calculation
```

Comment:
Shouldn't PRewardInference ... CNorm have `validateX` functions written for them?

Reply:
yeah, we need to add them!